server/api.go

```diff
@@ -34,6 +34,29 @@ const (
 func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 
+	if r.URL.Path == "/set-expiry" {
+		expiry := r.URL.Query().Get("expiry")
+		userId := r.Header.Get("Mattermost-User-Id")
+		token, err := p.getGitlabUserTokenByMattermostID(userId)
+		oldExpiry := token.Expiry
+		if err != nil {
+			http.Error(w, err.Error(), 500)
+			return
+		}
```

Comment:
We can use this to set the expiry of the user's token. Run this in the browser dev console to set the expiry to 1 minute ot 0 minutes:

```js
// set to 1 minute
fetch('/plugins/com.github.manland.mattermost-plugin-gitlab/set-expiry?expiry=1')
```

```js
// set to 0 minutes
fetch('/plugins/com.github.manland.mattermost-plugin-gitlab/set-expiry?expiry=0');


```